app/views/shared/_footer.html.erb

```diff
@@ -60,23 +60,23 @@
 
         <ol class="unstyled-list l-footer-social__list">
           <li class="l-footer-social__list-item twitter">
-            <a href="http://twitter.com/YourMoneyAdvice" title="Your Money Advice on Twitter">
+            <a href="http://twitter.com/YourMoneyAdvice" title="Your Money Advice on Twitter" class="footer-social__link">
```

Comment:
I've updated the class names for consistency. Not so sure about if adding a partial is a great improvement since these don't get shared anywhere.